apps/mymedicare_cb/views.py

```diff
@@ -105,14 +105,10 @@ def callback(request):
     response = requests.get(url, cert=certs, verify=False)
 
     if 'entry' in response.json():
-        identifiers = response.json()['entry'][0]['resource']['identifier']
-        fhir_id = ""
-        for i in identifiers:
-            if i['system'] == 'http://bluebutton.cms.hhs.gov/identifier#bene_id':
-                fhir_id = i['value']
-            if fhir_id:
-                cx.fhir_id = fhir_id
-                cx.save()
+        fhir_id = response.json()['entry'][0]['resource']['id']
```

Comment:
I'd probably add a check to ensure that the ID isn't empty. But I'm paranoid like that.